e2e/hpa/01-unmanage-hpa.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: quay.redhat.com/v1
+kind: QuayRegistry
+metadata:
+  name: hpa
+spec:
+  components:
+  - kind: clair
+    managed: false
+  - kind: clairpostgres
+    managed: false
+  - kind: base
+    managed: true
+  - kind: postgres
+    managed: true
+  - kind: redis
+    managed: true
+  - kind: horizontalpodautoscaler
+    managed: false
+  - kind: objectstorage
+    managed: true
+  - kind: route
+    managed: true
+  - kind: mirror
+    managed: true
+  - kind: monitoring
+    managed: true
+  - kind: tls
+    managed: true
+---
+apiVersion: autoscaling/v1
```

Comment:
Why do we create this HPA here ?

Reply:
> Why do we create this HPA here ?

I'm not sure what creation you're mentioning.

```
apiVersion: autoscaling/v1
kind: HorizontalPodAutoscaler
metadata:
  name: hpa-quay-mirror
spec:
  minReplicas: 1
  ```
  If you mean this, these lines update the existing hpa, existence of which is asserted in the first step.